src/commands.cpp

```diff
@@ -762,12 +844,12 @@ int Command::generate_launch_blob(uint32_t policy)
     memset(&session_data_buf, 0, sizeof(sev_session_buf));
 
     do {
-        // Read in the PDH (Platform Owner Diffie-Hellman Public Key)
+        // Read in the PDH (Platform Diffie-Hellman Public Key)
```

Comment:
Can we put all the "Platform Owner" stuff back, since we want to differentiate between the Platform Owner and Guest Owner? We want to differentiate between the guest owner and a guest, the same way that we want to differentiate between a platform owner and the platform itself.

Reply:
Just for my understanding, the PDH is a key that is stored inside of the SEV hardware, while its corresponding certificate (also oftentimes called PDH) can be exported using the `PDH_CERT_EXPORT` command? As such, its private key never leaves the SEV firmware and the public key can be used to establish a secure communication channel with the firmware? The platform owner is responsible for sharing its platform's PDH, PEK certificates and either CEK certificate or Chip-ID with a guest owner so that the guest owner can establish a secure communication channel? (I'm omitting the Owner Certificate Authority for simplicity)